Libs/Reflex/src/main/java/reflex/function/TimeNode.java

```diff
@@ -41,29 +44,40 @@
  */
 public class TimeNode extends BaseNode {
     private ReflexNode initExpression;
+    private List<ReflexNode> exprList;
 
-    public TimeNode(int lineNumber, IReflexHandler handler, Scope s, ReflexNode expression) {
-        super(lineNumber, handler, s);
-        this.initExpression = expression;
+    public TimeNode(int lineNumber, IReflexHandler handler, Scope scope, List<ReflexNode> eList) {
+        super(lineNumber, handler, scope);
+        if (eList == null) {
+            this.exprList = new ArrayList<>();
+        } else {
+            this.exprList = eList;
+        }
```

Comment:
thse 5 lines can be shortened to 3, if you just initialize the member variable to an empty array list and then only set it if it is not null